guides/release/components/helper-functions.md

```diff
@@ -315,7 +315,69 @@ components.
 ```
 
 In the component's template, you can then use the `person` object:
-
 ```handlebars {data-filename=app/components/greeting/template.hbs}
 Hello, {{@person.givenName}} {{@person.familyName}}
 ```
+
+### The `in-element` helper
+
+Using the [`{{in-element}}`](https://api.emberjs.com/ember/3.27/classes/Ember.Templates.helpers/methods/in-element?anchor=in-element) helper, you can render content into a specific DOM element that is in a different part of the page. For instance, we might want
+to render a modal, tooltip or dropdown.
+
+Let's say I wanted to show a dropdown when a button is clicked on.
+
+Below, we have the button we want to click on, a div element that will hold the dropdown and the dropdown component. We pass in `showDropdown` that will add the dropdown into our div.
+```handlebars {data-filename=app/components/some-component.hbs}
+  <button type="button" {{on "click" this.onClickShowDropdown}}>More Actions</button>
```

Comment:
To help with code readability, I recommend dedicating a line for each argument, etc.

```suggestion
  <button
    type="button"
    {{on "click" this.onClickShowDropdown}}
  >
    More Actions
  </button>

```

Reply:
To help with semantics and code maintenance, can we provide better names than `More Actions` and `onClickShowDropdown`?

I don't know the context that you had in mind (i.e. why would the user see `Archive`, `Mark as Read`, and `Report`?) to give a suggestion for the former.

For the latter, I could suggest `showDropdown` (this would require renaming the Boolean tracked property to `isDropdownShown`) and `insertDropdown`.

Reply:
Done. Yup this example is a dropdown.